tasks/examples.js

```diff
@@ -111,12 +111,25 @@ module.exports = function(gulp, config) {
 			.pipe(connect.reload());
 	});
 
-	gulp.task('build:example:css', function() {
-		return gulp.src(config.example.src + '/' + config.example.less)
-			.pipe(less())
-			.pipe(gulp.dest(config.example.dist))
-			.pipe(connect.reload());
-	});
+	function buildStylesheetsForExample() {
+		return function(){
+				return merge(
+				
+				gulp.src(config.example.src + '/' + config.example.less)
+					.pipe(less())
+					.pipe(gulp.dest(config.example.dist))
+					.pipe(connect.reload()),
+
+				gulp.src('./' + config.component.src + '/' + config.component.name + '.less')
```

Comment:
It isn't a given that there will be a file matching `'./' + config.component.src + '/' + config.component.name + '.less'`

That's the purpose of `config.component.less` - to specify the LESS file(s) that should be transformed.

In the case of examples, the less for the component is actually included from `./src` so there's no need to _also_ include the component less.